test/fw/ptl/utils/pbs_crayutils.py

```diff
@@ -48,30 +48,25 @@ class CrayUtils(object):
     """
     node_status = []
     node_summary = {}
+    cmd_output = []
     du = None
 
     def __init__(self):
         self.du = DshUtils()
         (self.node_status, self.node_summary) = self.parse_apstat_rn()
 
-    def parse_apstat_rn(self):
+    def call_apstat(self, options):
         """
-        Run apstat command on cray/craysim and parse its output
-
-        :param options: options to pass to apstat command
```

Comment:
I think you removed this line by mistake. Please add it back

Reply:
added 'param options' back